web-api/src/storage/elastic.rs

```diff
@@ -109,12 +114,48 @@ struct BulkItemResponse {
     error: Value,
 }
 
+impl BulkItemResponse {
+    fn is_success_status(&self, allow_not_found: bool) -> bool {
```

Comment:
this and the following function just moved up from the free-floating functions below